src/mame/nintendo/vt1682.cpp

```diff
@@ -72,11 +72,12 @@
 #include "screen.h"
 #include "speaker.h"
 
+#define LOG_DMA              (1U << 4)
 #define LOG_VRAM_WRITES      (1U << 1)
 #define LOG_SRAM_WRITES      (1U << 2)
 #define LOG_OTHER            (1U << 3)
```

Comment:
Please keep the bits used by the macros in ascending order for the order of declaration.  It’s a lot easier to see that there are no conflicts that way.